frontend/package.json

```diff
@@ -12,7 +12,8 @@
   "dependencies": {
     "prop-types": "^15.8.1",
     "react": "^18.0.0",
-    "react-dom": "^18.0.0"
+    "react-dom": "^18.0.0",
+    "template-fullstack": "file:.."
```

Comment:
this line looks like an error, and frontend/package.json and frontend/package-lock.json should not be modified by this PR : restore them